packages/dds/tree/api-report/tree.api.md

```diff
@@ -895,14 +886,11 @@ export interface IForestSubscription extends ISubscribable<ForestEvents> {
     tryMoveCursorToNode(destination: Anchor, cursorToMove: ITreeSubscriptionCursor): TreeNavigationResult;
 }
 
-// @internal
-export type ImplicitAllowedTypes = AllowedTypes | FlexTreeNodeSchema | Any;
-
-// @beta
-type ImplicitAllowedTypes_2 = AllowedTypes_2 | TreeNodeSchema;
+// @public
+export type ImplicitAllowedTypes = AllowedTypes | TreeNodeSchema;
 
-// @beta
-export type ImplicitFieldSchema = FieldSchema | ImplicitAllowedTypes_2;
+// @public
+export type ImplicitFieldSchema = FieldSchema | ImplicitAllowedTypes;
```

Comment:
Name collision between EditableTree2 'ImplicitAllowedTypes' and new 'class-schema' type.